project.clj

```diff
@@ -68,4 +68,6 @@
                                          merge-meta [[:inner 0]]}}}
 
              :eastwood {:plugins [[jonase/eastwood "0.2.3"]]
-                        :eastwood {:config-files ["eastwood.clj"]}}})
+                        :eastwood {:config-files ["eastwood.clj"]}}
+
+             :debug {:jvm-opts ["-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5009"]}})
```

Comment:
Ah, yes, unrelated to this PR. I put that in most of my projects so that I can attach Intellij to the running process and debug with Cursive. I can remove itl